tests/tests/control_wam_debug

```diff
@@ -0,0 +1,52 @@
+###############################################################################
+#
+#  Global control multigases debug test
+#
+###############################################################################
+
+export TEST_DESCR="Compare global control multigases debug results with previous trunk version"
+
+export CNTL_DIR=control_wam_debug
+
+export LIST_FILES="sfcf019.nc \
```

Comment:
I am curious, why does the forecast length set to 19?

Reply:
That took me a while to figure out ... the WAM test is a warmstart run. The REPRO run `control_wam` runs from fcst hr 18 to 24. Since debug runs take a lot of time, we typically do only 1h --> 18h to 19h